src/coreclr/src/System.Private.CoreLib/src/System/Globalization/GlobalizationMode.cs

```diff
@@ -6,18 +6,32 @@ namespace System.Globalization
 {
     internal static partial class GlobalizationMode
     {
+        // Order of these properties matter because GetUseIcuMode is dependent on Invariant.
+        // So we need Invariant to be initialized first.
         internal static bool Invariant { get; } = GetGlobalizationInvariantMode();
```

Comment:
```suggestion
        internal static bool Invariant { get; }
            = GetSwitchValue("System.Globalization.Invariant", "DOTNET_SYSTEM_GLOBALIZATION_INVARIANT");
```

This goes through two layers of wrappers. Why not just call it directly?

Reply:
Not for the .Unix.cs file. The .Unix.cs file has some logic before calling `GetInvariantSwitchValue`.

Reply:
I see what you’re saying now. I’ll fix it

Reply:
Ok, this should be easier to see https://github.com/dotnet/runtime/pull/34645#discussion_r405957915